src/ralph/api/__init__.py

```diff
@@ -14,12 +15,12 @@
 
 
 @lru_cache(maxsize=None)
-def get_health_check_routes():
+def get_health_check_routes() -> List:
     """Return the health check routes."""
     return [route.path for route in health.router.routes]
 
 
-def filter_transactions(event, hint):  # pylint: disable=unused-argument
+def filter_transactions(event: Any, hint) -> Any:  # pylint: disable=unused-argument
```

Comment:
Maybe we can be more specific?

```suggestion
def filter_transactions(event: Dict, hint) -> Dict:  # pylint: disable=unused-argument
```